# This file is part of MXE.
# See index.html for further information.

This patch has been taken from:
git clone git://git.code.sf.net/p/cygwin-ports/mingw64-x86_64-gcc

--- a/gcc/config/i386/mingw32.h	2013-01-10 14:38:27.000000000 -0600
+++ b/gcc/config/i386/mingw32.h	2013-03-14 03:45:34.220060900 -0500
@@ -245,4 +245,4 @@ do {						         \
 #define LIBGCC_SONAME "libgcc_s" LIBGCC_EH_EXTN "-1.dll"
 
 /* We should find a way to not have to update this manually.  */
-#define LIBGCJ_SONAME "libgcj" /*LIBGCC_EH_EXTN*/ "-13.dll"
+#define LIBGCJ_SONAME "libgcj" /*LIBGCC_EH_EXTN*/ "-14.dll"
